docs/source/tutorial/commercial_paper.md

```diff
@@ -573,7 +616,7 @@ separate terminal window for her, and in `fabric-samples` locate the MagnetoCorp
 (isabella)$ cd commercial-paper/organization/magnetocorp/application/
 (isabella)$ ls
 
-addToWallet.js		issue.js		package.json
+enrollUser.js		issue.js		package.json
 ```
```

Comment:
Right now, users will so both `enrollUsr.js` and `addToWallet.js` on their file system, but  `addToWallet.js`  will be removed in a follow on PR.